frontend/src/views/Lobby.tsx

```diff
@@ -639,6 +660,54 @@ function LobbyPage() {
                 />
               </SliderContainer>
             </HoverContainerSlider>
+            <NoMarginMediumText>Room Size</NoMarginMediumText>
+            <NoMarginSubtitleText>
+              {size === 31? 'No limit' : (size === 1? '1 person' : `${size} people`)}
+            </NoMarginSubtitleText>
+            <HoverContainerSlider>
+              <HoverElementSlider
+                enabled={isHost(currentUser)}
+                onMouseEnter={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(true);
+                  }
+                }}
+                onMouseLeave={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(false);
+                  }
+                }}
+              />
+              <SliderContainer>
+                <Slider
+                  min={1}
+                  max={31}
+                  value={size}
+                  disabled={!isHost(currentUser)}
+                  onChange={(e) => {
+                    const { value } = e.target;
+
+                    // Set duration to undefined to allow users to clear field
```

Comment:
Suggested change - however, when is it necessary to setSize to be undefined? 

```suggestion
                    // Set size to undefined to allow users to clear field
```